check/TestIis.cpp

```diff
@@ -42,10 +44,21 @@ TEST_CASE("lp-incompatible-bounds", "[iis]") {
   Highs highs;
   highs.setOptionValue("output_flag", dev_run);
   highs.passModel(lp);
+  // Perform the light IIS check
+  highs.setOptionValue("iis_strategy", kIisStrategyLight);
+  HighsIis iis;
+  REQUIRE(highs.getIis(iis) == HighsStatus::kOk);
```

Comment:
I could put it below, but shouldn't we capitalize IIS everywhere?
```suggestion
  REQUIRE(highs.getIIS(iis) == HighsStatus::kOk);
```

Reply:
No, with a couple of exceptions - created many years ago! - in HiGHS initialisms appear in camel case with only the first letter capitalised. So, we have HighsLp, for example. One never knows when something would be less readable in camel case if initialisms were all in upper case.